app/assets/javascripts/save_and_ingest_handler.es6

```diff
@@ -10,7 +10,7 @@ export default class SaveAndIngestHandler {
       this.button_element.val('Searching...')
       let change_set = window.location.pathname.split('/').pop()
       let qs = {}
-      if (change_set !== 'new') {
+      if (change_set !== 'new' || change_set !== 'edit') {
```

Comment:
Shouldn't this be `&&`?

Reply:
Totally. Was thinking "and" but typed out "or".

Reply:
Fixed.